### PR TITLE
Fixes slurring

### PR DESCRIPTION
**Pull request by adamsong**
# Document the changes in your pull request

Slurring was refactored by code stolen from tgstation/tgstation#66031 in #18514 but the actual slurring files were not stolen.

# Changelog

:cl:  
bugfix: fixed slurring
/:cl:

### CODE REVIEW

**Review by ToasterBiome — APPROVED** [review]
critical bugfix

**Comment by Hopekz** [comment]
> critical bugfix

It's been broken for 5 months

**Comment by ToasterBiome** [comment]
> > critical bugfix
> 
> It's been broken for 5 months

ok and

**Comment by Hopekz** [comment]
> > > critical bugfix
> > 
> > It's been broken for 5 months
> 
> ok and

Tl;Dr: only pointing out it wasn't critical.


Only matters because it would make us look bad if a "critical" fix took 5 months to get here.

Critical is a level usually reserved for something that impacts a major functionality to the point where the game can't be delivered without fixing the bug.

Basically by how we usually go about things and labeling this critical you are saying that the game has been unplayable for the last 5 months when this was broken.

**Comment by ToasterBiome** [comment]
> > > > critical bugfix
> > > 
> > > It's been broken for 5 months
> > 
> > ok and
> 
> Tl;Dr: only pointing out it wasn't critical.
> 
> 
> Only matters because it would make us look bad if a "critical" fix took 5 months to get here.
> 
> Critical is a level usually reserved for something that impacts a major functionality to the point where the game can't be delivered without fixing the bug.
> 
> Basically by how we usually go about things and labeling this critical you are saying that the game has been unplayable for the last 5 months when this was broken.

image

**Comment by Hopekz** [comment]
> > > > > critical bugfix
> > > > 
> > > > 
> > > > It's been broken for 5 months
> > > 
> > > 
> > > ok and
> > 
> > 
> > Tl;Dr: only pointing out it wasn't critical.
> > Only matters because it would make us look bad if a "critical" fix took 5 months to get here.
> > Critical is a level usually reserved for something that impacts a major functionality to the point where the game can't be delivered without fixing the bug.
> > Basically by how we usually go about things and labeling this critical you are saying that the game has been unplayable for the last 5 months when this was broken.
> 
> image

IMG_20230829_032332
